presto-native-execution/presto_cpp/main/types/PrestoToVeloxQueryPlan.cpp

```diff
@@ -147,6 +147,29 @@ std::shared_ptr<connector::hive::LocationHandle> toLocationHandle(
       toTableType(locationHandle.tableType));
 }
 
+dwio::common::FileFormat toFileFormat(
+    const protocol::HiveStorageFormat storageFormat) {
+  switch (storageFormat) {
+    case protocol::HiveStorageFormat::DWRF:
```

Comment:
Do we support all these formats now? @majetideepak @Yuhta Thanks!

Reply:
No, but we will get an informative reader factory error in velox anyway so it's actually good to forward them all here

Reply:
I removed them to keep it consistent with other similar conversions (eg. TableType TEMPORARY is not forwarded).
The error message is also cleaner in the protocol than a Velox factory error.